src/main/java/net/mcreator/element/converter/v2022_1/FoodToItemConverter.java

```diff
@@ -53,8 +53,8 @@ public GeneratableElement convert(Workspace workspace, GeneratableElement input,
 			item.renderType = food.get("renderType").getAsInt();
 			if (food.get("customModelName") != null)
 				item.customModelName = food.get("customModelName").getAsString();
-			item.creativeTab = new TabEntry(workspace,
-					food.get("creativeTab").getAsJsonObject().get("value").getAsString());
+			item.creativeTabs = List.of(new TabEntry(workspace,

```

Comment:
To my understanding, this converter will convert `"No creative tab entry"` to an entry in the list, which is not ok.

I recommend making a utility method in ItemsCreativeTabsConverter that you can call from other converters.

This applies to other converters too so please fix all of them and retest as much as possible again.

Reply:
The thing is the new converter already handles this case as intended, so easiest and shortest solution would be to just add the same check in other 2 as well.

Reply:
I asked for utility method, but given the code is not that long and in converters we accept a degree of duplicate, you can also do that.

But _please_, retest the PR again with as many cases as possible

Reply:
:heavy_check_mark: Fixed